pages/_app.js

```diff
@@ -142,7 +142,7 @@ function AccountSettingsPopup() {
 
   const evidence = evidenceURI ? getEvidenceFile(evidenceURI) : null;
   const displayName =
-    [evidence?.file.firstName, evidence?.file.lastName]
+    [evidence?.file?.firstName, evidence?.file?.lastName]
       .filter(Boolean)
       .join(" ") || evidence?.file.name;
```

Comment:
Shouldn't you change to `evidence?.file?` here as well?
Also, something to keep in mid, in case `evidence.file` is undefined, displayName will be undefined as well

Reply:
Added the conditional chaining to line 147. Regarding `displayName` being undefined is not a problem, is handled by the `AccountSettingsPopup`, if it's undefined it won't show.